reframe/core/config.py

```diff
@@ -528,6 +528,9 @@ def handler_list(handler_config):
                 f"by ReFrame based on '{filename}'.\n#\n\n"
                 f"site_configuration = {util.ppretty(converted)}\n")
 
+    contents = '\n'.join(l if len(l) < 80 else f'{l}  # noqa'
+                         for l in contents.split('\n'))
```

Comment:
Why not construct the `contents` above directly as a list so as not have to split it here? Also the line limit is 79 according to PEP8. Being pedantic, I would suggest to put the comment on the next multiple-of-indentation-depth column, if that does not make the code overly complicated.

Reply:
Also the annotation should be `# noqa: E501

Reply:
> Why not construct the contents above directly as a list so as not have to split it here?

You mean change `util.ppretty`? Do we want to return a list of lines, instead of the "pretty" string?

Reply:
Oh, ok you're right! I've just seen the content being set above as a string and didn't noticed the `ppretty`. Leave it as is then this part.

Reply:
if the next multiple-of-indentation position is separated by a single space from the end of the line, should we have the comment starting only after a single space? normally it's two spaces after.

Reply:
I think the limit is 79, including it https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
> if the next multiple-of-indentation position is separated by a single space from the end of the line, should we have the comment starting only after a single space? normally it's two spaces after.

The minimum should be two as the standard says. But we can keep it simple and just add two spaces and that's it.

> I think the limit is 79, including it https://www.python.org/dev/peps/pep-0008/#maximum-line-length

You have it right, you have used `< 80` instead of `<= 80`. That's fine.

Reply:
@rsarm Just fix the `noqa` annotation with the error code and we're ready to merge it.